extensions/resource-deployment/src/ui/resourceTypeWizard.ts

```diff
@@ -35,21 +40,30 @@ export class ResourceTypeWizard {
 		this._useGenerateScriptButton = value;
 	}
 
-	//TODO: eventually only resourceType will be passed. For now, we are passing both the resourceType and provider
+	//TODO: eventually only resourceType will be passed. For now, we are passing both \the resourceType and provider
 	constructor(
 		public resourceType: ResourceType,
-		public provider: DeploymentProvider,
 		public _kubeService: IKubeService,
 		public azdataService: IAzdataService,
 		public notebookService: INotebookService,
 		public toolsService: IToolsService,
-		public platformService: IPlatformService) {
-		this.wizardObject = azdata.window.createWizard(resourceType.displayName, resourceType.name, 'wide');
-		this.model = this.getResourceProviderModel()!;
+		public platformService: IPlatformService,
+		public resourceTypeService: ResourceTypeService) {
+
+		if (resourceType.options) {
+			let options: { option: string; value: string; }[] = [];
+			resourceType.options.forEach(option => {
```

Comment:
nit suggestion: use map instead of forEach as follows:

const options = resourceType.options.map(option => { option: option.name, value: option.values[0].name });